app/services/background.py

```
@@ -577,3 +586,133 @@ async def _cleanup_expired_sessions(self):
+    async def _automated_disk_cleanup(self):
+        """Automated disk cleanup for enterprise production environment."""
+        try:
+            self.logger.info("🧹 Starting automated disk cleanup")
+            cleanup_actions = []
+            
+            # 1. Clean up old log files (older than 7 days)
+            import os
+            import glob
+            from pathlib import Path
+            
+            log_dirs = ["/var/log", "./logs", "/tmp"]
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)